polkadot-parachains/integritee-runtime/src/weights/pallet_teerex.rs

```diff
@@ -39,11 +44,11 @@ impl<T: frame_system::Config> pallet_teerex::WeightInfo for WeightInfo<T> {
 	/// Proof Skipped: Teerex SovereignEnclaves (max_values: None, max_size: None, mode: Measured)
 	fn register_sgx_enclave() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `394`
-		//  Estimated: `3859`
-		// Minimum execution time: 2_019_887_000 picoseconds.
-		Weight::from_parts(2_047_667_000, 0)
-			.saturating_add(Weight::from_parts(0, 3859))
+		//  Measured:  `451`
+		//  Estimated: `3916`
+		// Minimum execution time: 2_821_014_000 picoseconds.
```

Comment:
I think that is close enough given different benchmarking machines

Reply:
Yes, I agree.